src/lib/tour-overlay/TourOverlay.component.tsx

```diff
@@ -61,24 +71,23 @@ export const TourOverlay = React.forwardRef<TourOverlayRef, TourOverlayProps>((p
     return spot.y + (spot.height / 2);
   }, [spot.y, spot.height]);
 
-  /**
-   * Animations in the native thread are disabled at the moment as they are
-   * causing insonconsistent renders due to the current animation workflow.
-   *
-   * We need to re-work how animation are executed, plus give the user the
-   * option to choose wheher or not the tour should use the native driver.
-   */
-  const useNativeDriver = useMemo(() => Platform.select({
-    android: false,
-    default: false,
-    ios: false
-  }), [Platform.OS]);
-
-  const getTipStyles = useCallback((tipLayout: LayoutRectangle): StyleProp<ViewStyle> => {
+  const useNativeDriver = useMemo(() => {
+    const driverConfig: OSConfig<boolean> = typeof nativeDriver === "boolean"
+      ? { android: nativeDriver, ios: nativeDriver }
+      : nativeDriver;
+
+    return Platform.select({
+      android: driverConfig.android,
+      default: false,
+      ios: driverConfig.ios
+    });
```

Comment:
Good work with the platform-specific config!

Reply:
🎉